msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerAccountManagerStrategy.java

```diff
@@ -426,8 +429,24 @@ protected boolean removeBrokerAccount(@NonNull final OperationParameters paramet
                         .putAction(methodName)
                         .isSuccessful(true)
         );
+    }
+
+    @Override
+    boolean getDeviceMode(@NonNull OperationParameters parameters) throws BaseException, InterruptedException, ExecutionException, RemoteException {
+        // TODO
```

Comment:
any reason why these are not implemented ? this should be same as if it's going through bind service right?

Reply:
This will be a separate card - they're not implemented yet.